auxprogs/joingenes/Makefile

```diff
@@ -1,20 +1,20 @@
 CC=g++
-CFLAGS=-Wall -std=gnu++0x
+CXXFLAGS+=-Wall -std=gnu++0x
 
 all: joingenes
 
 joingenes: joingenes.o jg_transcript.o jg_ios.o jg_transcript.h jg_ios.h
-	$(CC) $(CFLAGS) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes $(LDFLAGS)
 	cp joingenes ../../bin/
 
 joingenes.o: joingenes.cpp jg_transcript.h jg_ios.h
-	$(CC) -c $(CFLAGS) $(CXXFLAGS) $(CPPFLAGS) ${LDFLAGS} joingenes.cpp
+	$(CC) -c $(CPPFLAGS) $(CXXFLAGS) joingenes.cpp
```

Comment:
LDFLAGS no longer needed : why this flag is used in the Makefiles before on similar locations?